mscore/editlyrics.cpp

```diff
@@ -36,8 +37,12 @@ bool ScoreView::editKeyLyrics()
                   if (!editData.control(textEditing)) {
                         if (editData.s == "_")
                               lyricsUnderscore();
-                        else // TODO: shift+tab events are filtered by qt
-                              lyricsTab(editData.modifiers & Qt::ShiftModifier, true, false);
+                        else { // TODO: shift+tab events are filtered by qt
+                              Shortcut* mySC = Shortcut::getShortcut("prev-lyric");
+                              for (auto &ks : mySC->keys())
```

Comment:
The `&` should be put before the space. So `auto& ks`. This is in MuseScore coding rules. It may be changed, but let's just follow the rules for now.